src/cdk/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -91,6 +90,7 @@ describe('CdkVirtualScrollViewport', () => {
       spyOn(viewport, 'checkViewportSize').and.callThrough();
 
       dispatchFakeEvent(window, 'resize');
+      fixture.changeDetectorRef.markForCheck();
```

Comment:
This feels suspicious. What's supposed to happen here with the resize event? I'm guessing users aren't expected to call `markForCheck` on resize.

Same question for the other calls to `markForCheck` after triggering the scroll event.